.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Node CI
```

Comment:
Maybe `Build`?
Also, please rename the file.
```suggestion
name: Build
```

Reply:
Fixed